### ISSUE TITLE
Issue with aircannon item

### ISSUE THREAD

**Issue by Floodo** [srnyx/air-cannon#1]
### Expected behavior

The aircannon item should boost me forwards/backwards

### Actual behavior

It does absolutely nothing.

Config file:
https://paste.venox.network/ozeyocolit.yml

### Steps to reproduce

Well it just happend after shutting down a server and than starting it up. Any other player could use it normaly, but I can't (im OP player). Everytime I wwanted to use that item, it gives me warn messages in console (https://paste.venox.network/sezoyokabu.txt). After destroying that item and giving myself a new one, it was okay (but what if it will happend again?).

### Version

Paper server v1.19

### Checks

- [X] I asked in the Discord to see whether this issue is in fact a bug that needs to be fixed.

**Comment by srnyx** [comment]
Did you change anything in the config or update the plugin between shutting down and starting the server?

**Comment by Floodo** [comment]
nope, since I updated to the newest version of this plugin I didn't change anything.

**Comment by srnyx** [comment]
> nope, since I updated to the newest version of this plugin I didn't change anything.

Did you update Minecraft versions? What other plugins do you have installed (and of them, which might modify items)?
I don't really know what could've caused this. So if it was none of those things I'll have to do some more research on my end.

**Comment by Floodo** [comment]
Well I have  the last Paper server v1.19 and also MC version 1.19. Well I have a lot of plugins :DD .... but none of them is item modifying. If you really want I can screenshot the list of all plugins.

**Comment by srnyx** [comment]
stale and not reproducible
